SU2_CFD/src/python_wrapper_structure.cpp

```diff
@@ -381,14 +381,14 @@ bool CDriver::ComputeVertexForces(unsigned short iMarker, unsigned short iVertex
     Area = sqrt(AreaSquare);
 
     /*--- Get the values of pressure and viscosity ---*/
-    Pn = solver_container[ZONE_0][INST_0][MESH_0][FLOW_SOL]->node[iPoint]->GetPressure();
+    Pn = solver_container[ZONE_0][INST_0][MESH_0][FLOW_SOL]->node->GetPressure(iPoint);
     if (viscous_flow) {
       for(iDim=0; iDim<nDim; iDim++) {
         for(jDim=0; jDim<nDim; jDim++) {
-          Grad_Vel[iDim][jDim] = solver_container[ZONE_0][INST_0][FinestMesh][FLOW_SOL]->node[iPoint]->GetGradient_Primitive(iDim+1, jDim);
+          Grad_Vel[iDim][jDim] = solver_container[ZONE_0][INST_0][FinestMesh][FLOW_SOL]->node->GetGradient_Primitive(iPoint, iDim+1, jDim);
```

Comment:
If it stays like that we could just remove the `node` ?

Reply:
The interface to CSolver is already pretty big, if we concatenate it with the one for CVariable it is possibly going to get a bit confusing.

Reply:
Yep I'll agree actually ... lets rename that to "nodes" then ?

Reply:
It is the least disruptive option, what should I do with the new `snode` I added? `snodes` ? `Snodes` to make clear it is not a typo?

Reply:
Or maybe it would be better to use a function called `GetNodes` to access the nodes from the outside. Internally we use then always the `snode` which we rename to be `nodes`. That is maybe less confusing ? The current `node` can then be something descriptive to make clear its not intended to be used internally.

Reply:
@talbring if we add a `GetNodes` function the best thing is to keep only a private "`snode`" in each solver which that function upcasts to CVariable, no room for doubt that way.

@economon being the solution to a PDE is what made me think of `fields` I am partial to short names otherwise it will mess with the code formatting too much.
IDK if something as descriptive as eulerVariables is needed, it is going to look weird in the NS solver and the IDE gives you that information already as it is the name of the variable type.

Reply:
Then let's go with function and a cast I'd say.

Reply:
And what if we call it `nodeArray` or `nodeData` and the class e.g. `CEulerData`?

Reply:
I agree with @talbring we need to use GetNodes to access it. As for the naming, fields or DOF  might be too cryptic I think. Data is also confusing (parameter or solution?). What about SolutionNode or SolNode or SolNodeFlow? We can then have SolNodeEul and SolNodeNS or if you prefer SolNodeAdv (for Advection) and SolNodeVis (Viscous) in the pertinent solvers. To be clear, SolNodeVis should contain all variables, including density and velocity.

Reply:
We are bikeshedding, so here is what I can reasonably do:
- Rename `node` to `vars` or `nodes` so I don't have to reformat 5k lines of code by hand, both make clear it's multiple things, but `vars` is consistent with the class names, to which I can append an "s" (CVariables and so on).
- Put a pure virtual access method in all solvers to avoid ambiguity between `node` and `snode` within each solver class, the name for that is `GetVars` or `GetNodes`.

Take it or leave it :)

Reply:
Once we are already at it, lets do it right. I can also help.

Anything with sol(ution) in it is confusing because it contains more than just that. Still thinking `data` would be the best, as that's what it is. 

Accessing it would be then:
`solver->GetNodeData()->GetSolution(iPoint, iVar)`